package.json

```diff
@@ -15,8 +15,8 @@
     "url": "https://github.com/andrewkeig/express-validation"
   },
   "dependencies": {
-    "@types/joi": "^13.0.5",
-    "lodash": "^4.9.0"
+    "lodash": "^4.17.15",
```

Comment:
Why did these reverse order and become un-alphabetised?

Reply:
@isaachinman  I have fixed the order of lodash.